Gemfile.lock

```diff
@@ -7,9 +7,9 @@ GEM
       rexml
     kramdown-parser-gfm (1.1.0)
       kramdown (~> 2.0)
-    kramdown-rfc (1.6.1)
-      kramdown-rfc2629 (= 1.6.1)
-    kramdown-rfc2629 (1.6.1)
+    kramdown-rfc (1.6.2)
+      kramdown-rfc2629 (= 1.6.2)
+    kramdown-rfc2629 (1.6.2)
```

Comment:
This `Gemfile.lock` specifies `PLATFORMS` of "arm64-darwin-20".  Should this include "x86_64-linux"?

Reply:
By default `bundler` adds the platform it runs on, this shouldn't be an issue as long as we don't have any platform-specific Gems.
I'll add `x86_64-linux` with #86.